dazel.py

```diff
@@ -188,15 +188,18 @@ def send_command(self, args):
             return os.WEXITSTATUS(rc)
 
     def _fix_win_symlink(self, docker_exec_command):
-        p = pathlib.Path(self.directory)
-        for path in list(p.glob('bazel-*')):
-            command = "%s realpath %s" % (
-                docker_exec_command, str(path.name))
-            output = self._run_command(command).strip()
-            drive = pathlib.PureWindowsPath(path.drive)
-            local_directory = drive.joinpath(pathlib.PurePosixPath(output))
-            path.unlink()
-            path.symlink_to(local_directory, target_is_directory=True)
+        try:
+            p = pathlib.Path(self.directory)
+            for path in list(p.glob('bazel-*')):
+                command = "%s realpath %s" % (
+                    docker_exec_command, str(path.name))
+                output = self._run_command(command).strip()
+                drive = pathlib.PureWindowsPath(path.drive)
+                local_directory = drive.joinpath(pathlib.PurePosixPath(output))
+                path.unlink()
+                path.symlink_to(local_directory, target_is_directory=True)
+        except subprocess.CalledProcessError:
```

Comment:
Actually, can you maybe put the try/except around the `output = self._run_command(command).strip()` line to make it clearer what it does?

Also - they way you did it now, if the first symlink exists, but the rest don't, it'll skip creating the other ones, which is not what you want.

Reply:
I moved inside the for loop, you are right about that we shall not skip the other links just because the first failed.
Regarding try/except around output, it is not possible if we fail here we cannot continue doing the other steps since they depend on the output.
But good comment!